<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongsTagScreen</class>
 <widget class="QWidget" name="SongsTagScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>160</width>
     <height>153</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="top_w" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="columns_w" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
